storage/include/rocksdb/client.h

```diff
@@ -44,7 +44,7 @@ class ClientIterator : public concord::storage::IDBClient::IDBClientIterator {
   KeyValuePair next() override;
   KeyValuePair getCurrent() override;
   bool isEnd() override;
-  concordUtils::Status getStatus() override;
```

Comment:
I think if we rely that the `concordUtils` namespace or some if its types are accessible in the current header, it hides bringing it up explicitly in some other header file.